<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Add</class>
 <widget class="QWidget" name="Add">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="returnButton">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>300</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>主菜单</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>300</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>添加</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="idLineEdit">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>50</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="idLabel">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>50</y>
     <width>72</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>学号</string>
   </property>
  </widget>
	 <widget class="QLabel" name="nameLabel">
		 <property name="geometry">
			 <rect>
				 <x>200</x>
				 <y>90</y>
				 <width>72</width>
				 <height>21</height>
			 </rect>
		 </property>
		 <property name="text">
			 <string>姓名</string>
		 </property>
	 </widget>
	 <widget class="QLineEdit" name="nameLineEdit">
		 <property name="geometry">
			 <rect>
				 <x>300</x>
				 <y>90</y>
				 <width>113</width>
				 <height>21</height>
			 </rect>
		 </property>
	 </widget>
	 <widget class="QLabel" name="gradeLabel">
		 <property name="geometry">
			 <rect>
				 <x>200</x>
				 <y>130</y>
				 <width>72</width>
				 <height>21</height>
			 </rect>
		 </property>
		 <property name="text">
			 <string>班级</string>
		 </property>
	 </widget>
	 <widget class="QLineEdit" name="gradeLineEdit">
		 <property name="geometry">
			 <rect>
				 <x>300</x>
				 <y>130</y>
				 <width>113</width>
				 <height>21</height>
			 </rect>
		 </property>
	 </widget>
  <widget class="QLabel" name="ALabel">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>170</y>
     <width>72</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>科目A</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ALineEdit">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>170</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="BLabel">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>210</y>
     <width>72</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>科目B</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="BLineEdit">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>210</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="CLabel">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>250</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>科目C</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="CLineEdit">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>250</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
